tensorflow/python/keras/layers/core.py

```diff
@@ -140,6 +140,11 @@ class Dropout(Layer):
   (This is in contrast to setting `trainable=False` for a Dropout layer.
   `trainable` does not affect the layer's behavior, as Dropout does
   not have any variables/weights that can be frozen during training.)
+  
+   Usage Example:
+   model = tf.keras.models.Sequential()
+   model.add(tf.keras.layers.Dropout(0.2, input_shape=(16,)))
+   model.add(tf.keras.layers.Dense(1))
```

Comment:
This doesn't follow the proper syntax at https://www.tensorflow.org/community/contribute/docs. Should be

```suggestion
   Usage Example:
   
   >>> model = tf.keras.models.Sequential()
   >>> model.add(tf.keras.layers.Dropout(0.2, input_shape=(16,)))
   >>> model.add(tf.keras.layers.Dense(1))
```